pkg/aks/components/AksNodePool.vue

```diff
@@ -215,7 +224,7 @@ export default defineComponent({
           type="number"
           :mode="mode"
           label-key="aks.nodePools.count.label"
-          :disabled="pool.enableAutoScaling"
+          :rules="validationRules.count"
```

Comment:
would it be a good idea to also add `min="1"` here?

Reply:
Good call. I added min and max attributes to node count, and node min/max inputs. It looks like that attribute stops the user from lowering the value below that point with the arrow buttons but its still possible to manually type in a lower number, so I have left in the form validation rules.

Reply:
You're right, it's also possible for the user to inspect the page's html and manually remove the attribute, but I think it's ok since we're just improving the ux here, and as you mentioned we already have the form validation in place.